docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -29,17 +30,30 @@ All other prerequisites are handled by the installed tools listed above. The sam
 
 You will need storage for the backups. This topic shows the use of AWS S3 buckets.
 
-* If you'll use AWS S3, before proceeding with the configuration described below, verify that you know the `aws_access_key_id` and `aws_secret_access_key` values. Or  contact your IT team if they manage those assets. You'll provide those details in an edited version of the [medusa-bucket-key.yaml](./medusa-bucket-key.yaml) file. For information about the S3 setup steps, see this helpful [readme](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md).  
+* If you'll use AWS S3, before proceeding with the configuration described below, verify that you know the `aws_access_key_id` and `aws_secret_access_key` values. Or  contact your IT team if they manage those assets. You'll provide those details in an edited version of the [medusa-bucket-key.yaml](medusa-bucket-key.yaml) file. For information about the S3 setup steps, see this helpful [readme](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md).  
+
+* Add and update the following repo, which has in one chart all the settings for K8ssandra plus the backup/restore settings:
+
+```
+helm repo add k8ssandra https://helm.k8ssandra.io/
+helm repo update
+```
+
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "k8ssandra" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
 
 ### Create secret for read/write access to an S3 bucket
 
-Before creating the k8ssandra cluster, we need to supply credentials so that Medusa has read/write to an S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, GKE, and other bucket types. In this example, we’re using S3.
+Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write to an AWS S3 bucket, which is where the backup will be stored.  Currently, Medusa supports local, Amazon S3, Google Cloud Storage, and Azure buckets. Currently, K8ssandra supports S3. 
```

Comment:
```suggestion
Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write to an AWS S3 bucket, which is where the backup will be stored.  Medusa currently supports local, Amazon S3, Google Cloud Storage, and Azure buckets. K8ssandra only exposes configuration for Amazon S3 at this time.
```

Reply:
There are couple problems I spotted. Here is a revised version:

Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write access to an AWS S3 bucket, which is where the backup will be stored.  Medusa currently supports local, Amazon S3, Google Cloud Storage, and Azure Blob Storage.. K8ssandra exposes configuration for Amazon S3 and Google Cloud Storage at this time.

Reply:
Updated suggestion with @jsanda's recommendation

```suggestion
Before installing the k8ssandra cluster, we need to supply credentials so that Medusa has read/write access to an AWS S3 bucket, which is where the backup will be stored. Medusa currently supports local, Amazon S3, Google Cloud Storage, Azure Blob Storage, and IBM Cloud Object Storage. K8ssandra exposes configuration for Amazon S3 and Google Cloud Storage at this time.
```

Reply:
@johnsmartco if this works for you commit the suggestion and we will wrap up approvals. Thanks!